substrate/frame/revive/src/lib.rs

```diff
@@ -1737,6 +1760,12 @@ sp_api::decl_runtime_apis! {
 			key: Vec<u8>,
 		) -> GetStorageResult;
 
+		/// Get all storage keys for a specified contract.
+		///
+		/// Returns `Ok(Vec<Vec<u8>>)` if the contract exists, containing all storage keys
+		/// associated with the contract. Otherwise returns `Err(ContractAccessError)`
+		fn get_storage_keys(address: H160) -> Result<Vec<Vec<u8>>, ContractAccessError>;
```

Comment:
Feels weird that both this function and the one from here https://github.com/paritytech/polkadot-sdk/pull/9617/files, needs to be added to the public APIs just for forge test,

Would it make more sense to actually have something like ReviveAPIExtension, ReviveAPITestExtension trait and have all of them there ?

Reply:
Nit I would just use an Option here

Maybe we should use a BoundedVec just to be safe, so that this can not be exploited :thinking: